Source/Mosa.Kernel.BareMetal/HAL/Hardware.cs

```diff
@@ -70,7 +70,7 @@ public override void DebugWriteLine(string message)
 	/// <param name="message">The message.</param>
 	public override void Abort(string message)
 	{
-		Debug.Fatal(message);
+		Debug.Fatal(message ?? "Abort");
```

Comment:
Is the null fallback needed, since the message isn't nullable?

Reply:
Just in case NULL is passed.